pkg/mls/api/v1/service.go

```diff
@@ -174,8 +174,8 @@ func (s *Service) SendGroupMessages(ctx context.Context, req *proto.SendGroupMes
 		}
 
 		// TODO: Wrap this in a transaction so publishing is all or nothing
-		wakuTopic := topic.BuildGroupTopic(result.GroupId)
-		msg, err := s.store.InsertGroupMessage(ctx, result.GroupId, data)
+		wakuTopic := topic.BuildGroupTopic([]byte(result.GroupId))
+		msg, err := s.store.InsertGroupMessage(ctx, []byte(result.GroupId), data)
```

Comment:
I actually hex encode the group_id in the validation service. I should probably unwind that decision and just make it return the bytes, but until then you can hex decode it here to get the original value. When we built the service we thought we'd want to treat group_ids as strings.

I'll take care of making a PR to change the validation service return type to bytes once all of this has landed.